### PR TITLE
Changed some messages

### PR DESCRIPTION
**Pull request by c-bordon**
|Related issue|
|---|
|closes https://github.com/wazuh/wazuh-packages/issues/1436|

## Description

Various messages are updated based on what was requested in the issue

## Logs example

```
06/04/2022 20:23:05 INFO: Starting Wazuh installation assistant. Wazuh version: 4.3.0
06/04/2022 20:23:05 INFO: Verbose logging will be redirected to /var/log/wazuh-install.log
06/04/2022 20:23:05 WARNING: Hardware and system checks ignored.
06/04/2022 20:23:16 INFO: --- Dependencies ----
06/04/2022 20:23:16 INFO: Installing apt-transport-https.
06/04/2022 20:23:19 INFO: Installing unzip.
06/04/2022 20:23:26 INFO: Wazuh development repository added.
06/04/2022 20:23:26 INFO: --- Configuration files ---
06/04/2022 20:23:26 INFO: Generating configuration files.
06/04/2022 20:23:27 INFO: Created wazuh-install-files.tar. It contains the Wazuh cluster key, certificates, and passwords necessary for installation.
06/04/2022 20:23:27 INFO: --- Wazuh indexer ---
06/04/2022 20:23:27 INFO: Starting Wazuh indexer installation.
06/04/2022 20:24:26 INFO: Wazuh indexer installation finished.
06/04/2022 20:24:26 INFO: Wazuh indexer post-install configuration finished.
06/04/2022 20:24:26 INFO: Starting service wazuh-indexer.
06/04/2022 20:24:36 INFO: wazuh-indexer service started.
06/04/2022 20:24:36 INFO: Initializing Wazuh indexer cluster security settings.
06/04/2022 20:24:41 INFO: Wazuh indexer cluster initialized.
06/04/2022 20:24:41 INFO: --- Wazuh server ---
06/04/2022 20:24:41 INFO: Starting the Wazuh manager installation.
06/04/2022 20:25:28 INFO: Wazuh manager installation finished.
06/04/2022 20:25:28 INFO: Starting service wazuh-manager.
06/04/2022 20:25:43 INFO: wazuh-manager service started.
06/04/2022 20:25:43 INFO: Starting Filebeat installation.
06/04/2022 20:25:52 INFO: Filebeat installation finished.
06/04/2022 20:25:53 INFO: Filebeat post-install configuration finished.
06/04/2022 20:25:53 INFO: Starting service filebeat.
06/04/2022 20:25:54 INFO: filebeat service started.
06/04/2022 20:25:54 INFO: --- Wazuh dashboard ---
06/04/2022 20:25:54 INFO: Starting Wazuh dashboard installation.
06/04/2022 20:26:35 INFO: Wazuh dashboard installation finished.
06/04/2022 20:26:35 INFO: Wazuh dashboard post-install configuration finished.
06/04/2022 20:26:35 INFO: Starting service wazuh-dashboard.
06/04/2022 20:26:36 INFO: wazuh-dashboard service started.
06/04/2022 20:26:50 INFO: Initializing Wazuh dashboard web application.
06/04/2022 20:27:01 INFO: Wazuh dashboard web application initialized.
06/04/2022 20:27:01 INFO: --- Summary ---
06/04/2022 20:27:01 INFO: You can access the web interface https://<wazuh-dashboard-ip>.
    User: admin
    Password: g4OTrDTfknI6uGisYRsLS3k9tCld4u4c
06/04/2022 20:27:01 INFO: Remember that the password can be seen in the file /var/log/wazuh-install.log
06/04/2022 20:27:01 INFO: Installation finished.
```

## Tests

Centos: https://devel.ci.wazuh.info/view/Tests/job/Test_unattended/517/console
Ubuntu: https://devel.ci.wazuh.info/view/Tests/job/Test_unattended/518/console
Distributted: https://devel.ci.wazuh.info/view/Tests/job/Test_unattended_distributed/421/console

### CODE REVIEW

File: unattended_installer/install_functions/dashboard.sh [hunk 1]
```diff
@@ -121,12 +121,14 @@ function dashboard_initialize() {
             exit 1
         else
             common_logger "--- Summary ---"
-            common_logger "When Wazuh dashboard is able to connect to your Wazuh indexer cluster, you can access the web interface https://${nodes_dashboard_ip}.\n    User: admin\n    Password: ${u_pass}"
+            common_logger "When Wazuh dashboard is able to connect to your Elasticsearch cluster, you can access the web interface https://${nodes_dashboard_ip}.\n    User: admin\n    Password: ${u_pass}"
+            common_logger "Remember that the password can be seen in the file ${logfile}"
```
Comment (c-bordon) [inline unattended_installer/install_functions/dashboard.sh:125]:
```suggestion
            common_logger "The password can be seen in the file ${logfile}"
```

File: unattended_installer/install_functions/dashboard.sh [hunk 2]
```diff
@@ -121,12 +121,14 @@ function dashboard_initialize() {
...
         fi
     else
         common_logger "Wazuh dashboard web application initialized."
         common_logger "--- Summary ---"
         common_logger "You can access the web interface https://${nodes_dashboard_ip}.\n    User: admin\n    Password: ${u_pass}"
+        common_logger "Remember that the password can be seen in the file ${logfile}"
```
Comment (c-bordon) [inline unattended_installer/install_functions/dashboard.sh:131]:
```suggestion
        common_logger "The password can be seen in the file ${logfile}"
```

**Review by c-bordon — COMMENTED** [review]
Some Changes

File: unattended_installer/install_functions/dashboard.sh [hunk 3]
```diff
@@ -121,12 +121,14 @@ function dashboard_initialize() {
             exit 1
         else
             common_logger "--- Summary ---"
-            common_logger "When Wazuh dashboard is able to connect to your Wazuh indexer cluster, you can access the web interface https://${nodes_dashboard_ip}.\n    User: admin\n    Password: ${u_pass}"
+            common_logger "When Wazuh dashboard is able to connect to your Elasticsearch cluster, you can access the web interface https://${nodes_dashboard_ip}.\n    User: admin\n    Password: ${u_pass}"
```
Comment (alberpilot) [inline unattended_installer/install_functions/dashboard.sh:124]: Why Elasticsearch cluster?

File: unattended_installer/install_functions/installMain.sh [hunk 4]
```diff
@@ -207,7 +207,7 @@ function main() {
     fi
 
     common_logger "Starting Wazuh installation assistant. Wazuh version: ${wazuh_version}"
-    common_logger "Verbose logging redirected to ${logfile}"
+    common_logger "Verbose logging will be redirected to ${logfile}"
```
Comment (alberpilot) [inline unattended_installer/install_functions/installMain.sh:210]: I think it's better not to change that according to content team advice: do not promise, just do it.

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: unattended_installer/install_functions/dashboard.sh (hunk 3 above)
Comment (c-bordon) [reply unattended_installer/install_functions/dashboard.sh]: Done: https://github.com/wazuh/wazuh-packages/pull/1440/commits/fe9c6f8f6f7d0532bcbf75230fd5ba819a092e48

File: unattended_installer/install_functions/installMain.sh (hunk 4 above)
Comment (c-bordon) [reply unattended_installer/install_functions/installMain.sh]: Done: https://github.com/wazuh/wazuh-packages/pull/1440/commits/fe9c6f8f6f7d0532bcbf75230fd5ba819a092e48

**Review by alberpilot — APPROVED** [review]

**Pull request merged**